extensions/ql-vscode/src/databases/ui/db-panel.ts

```diff
@@ -266,6 +280,85 @@ export class DbPanel extends DisposableObject {
     await this.dbManager.setSelectedDbItem(treeViewItem.dbItem);
   }
 
+  private async renameItem(treeViewItem: DbTreeViewItem): Promise<void> {
+    const dbItem = treeViewItem.dbItem;
+    if (dbItem === undefined) {
+      throw new Error(
+        "Not a database item that can be renamed. Please select a valid item.",
+      );
+    }
+
+    const newName = await window.showInputBox({
+      prompt: "Enter the new name",
+    });
```

Comment:
This is more of a usability comment than a code comment:

Would it be reasonable to put the old name as the preset value in the input box? I feel like that's more similar to a "normal" renaming experience (e.g. when you rename something in the file explorer).

Reply:
Thanks! I was being lazy not implementing this. Done now.